tools/mo/requirements.txt

```diff
@@ -11,3 +11,4 @@ urllib3>=1.26.4
 requests>=2.25.1
 fastjsonschema~=2.15.1
 openvino-telemetry>=2022.1.0
+pydot>=1.4.1,<2.0.0
```

Comment:
why is it needed?

Reply:
because we want to avoid testing everything with pydot 2.0, which is a big update on their side, that may affect us

Reply:
discussed offline